Makefile

```diff
@@ -366,6 +366,7 @@ $(eval $(call build-package,py3-s3transfer,0.5.1-r0))
 $(eval $(call build-package,py3-urllib3,1.26.13-r0))
 $(eval $(call build-package,py3-yaml,6.0-r0))
 $(eval $(call build-package,aws-cli,1.27.38-r0))
+$(eval $(call build-package,kubescape,2.0.182-r0))
```

Comment:
```suggestion
$(eval $(call build-package,kubescape,2.0.183-r0))
```

Reply:
thanks @dwertent, I've upgraded it accordingly.